Rakefile

```diff
@@ -4,6 +4,7 @@
 require File.expand_path('../config/application', __FILE__)
 require 'rake'
 require 'ci/reporter/rake/minitest' if Rails.env.test?
+require 'ci/reporter/rake/rspec'
```

Comment:
Because the ci-reporter gem is only available in the test gem group, you need to add the same guard clause to this new line as above.  Otherwise you block any rake tasks run on deployed environments where we don't install the test gem group.